docs/01-Introduction.md

```diff
@@ -3,3 +3,10 @@
 ## 1.1 What is LORIS?
 
 ## 1.2 How does LORIS work?
+
+
+### Deidentification of imaging datasets
```

Comment:
I would ask @davidblader to add this to his pull request (https://github.com/aces/Loris-MRI/pull/204) as he can integrate it better with the text he has?